src/main/kotlin/venus/glue/Renderer.kt

```diff
@@ -166,7 +168,17 @@ internal object Renderer {
      */
     fun updateRegister(id: Int, value: Int, setActive: Boolean = false) {
         val register = getElement("reg-$id-val") as HTMLInputElement
-        register.value = toHex(value)
+        var shown_val: String
```

Comment:
Kotlin supports [the `when` block](https://kotlinlang.org/docs/reference/control-flow.html#when-expression), so this could be more cleanly written as:

    register.value = when (displayType) {
        "hex" -> toHex(value)
        "dec" -> value.toString()
        "unsign" -> toUnsigned(value)
        else -> toHex(value)
    }